tests/aws/services/events/test_events_targets.py

```diff
@@ -26,111 +30,178 @@
     TEST_LAMBDA_PYTHON_ECHO,
 )
 
-
 # TODO:
 #  These tests should go into LocalStack Pro:
 #   - AppSync (pro)
 #   - Batch (pro)
 #   - Container (pro)
 #   - Redshift (pro)
 #   - Sagemaker (pro)
-class TestEventsTargetCloudWatchLogs:
-    @markers.aws.validated
-    def test_put_events_with_target_cloudwatch_logs(
-        self,
-        events_create_event_bus,
-        events_put_rule,
-        events_log_group,
-        aws_client,
-        snapshot,
-        cleanups,
-    ):
-        snapshot.add_transformers_list(
-            [
-                snapshot.transform.key_value("EventId"),
-                snapshot.transform.key_value("RuleArn"),
-                snapshot.transform.key_value("EventBusArn"),
-            ]
-        )
 
-        event_bus_name = f"test-bus-{short_uid()}"
-        event_bus_response = events_create_event_bus(Name=event_bus_name)
-        snapshot.match("event_bus_response", event_bus_response)
 
-        log_group = events_log_group()
-        log_group_name = log_group["log_group_name"]
-        log_group_arn = log_group["log_group_arn"]
-
-        resource_policy = {
-            "Version": "2012-10-17",
-            "Statement": [
-                {
-                    "Sid": "EventBridgePutLogEvents",
-                    "Effect": "Allow",
-                    "Principal": {"Service": "events.amazonaws.com"},
-                    "Action": ["logs:CreateLogStream", "logs:PutLogEvents"],
-                    "Resource": f"{log_group_arn}:*",
-                }
-            ],
-        }
-        policy_name = f"EventBridgePolicy-{short_uid()}"
-        aws_client.logs.put_resource_policy(
-            policyName=policy_name, policyDocument=json.dumps(resource_policy)
-        )
+class TestEventsTargetApiDestination:
+    # TODO validate against AWS
```

Comment:
this pr is only refactoring no changes per see - also would be great if you can do this since you are now more familiar with this test and API endpoints